dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/watchdog/ForcedMaxOutputRowsRule.scala

```diff
@@ -49,8 +50,12 @@ case class ForcedMaxOutputRowsRule(session: SparkSession) extends Rule[LogicalPl
 
     maxOutputRowsOpt match {
       case Some(forcedMaxOutputRows) => val supported = p match {
+          case agg: Aggregate => agg match {
+            case Aggregate(_, Alias(_, "havingCondition")::Nil, _) => false
```

Comment:
can we code ?
```
case Filter(_, _: Aggregate) =>
case _: Aggregate =>
```

Reply:
It's difficulty to process the situation according the order , because aggregate function will be called nested